.github/workflows/update-go-dependencies.yaml

```diff
@@ -0,0 +1,30 @@
+name: Update go dependencies and create PR
+
+on:
+  repository_dispatch:
+    types: update-dependencies
+  schedule:
+    - cron:  '0 12 * * 1'
```

Comment:
We should keep this aligned with the release job, that dependencies are updates shortly after the release. 
A comment about this dependency would be good.

Reply:
Yes, I agree. Added a comment.